### DISCUSSION TITLE
A problem of using BinghamODF function

### DISCUSSION THREAD

**Discussion by caoyi0701** [mtex-toolbox/mtex#1689] [Ask Anything]
Hi MTEX community,

I met a problem when I use Bingham ODF function to calculate a fiber odf.

I ran the following code.
`cs = crystalSymmetry('12/m1');
ss = specimenSymmetry('-1');
kappa = [10 10 -10 -10];   % shape parameters
%U     = eye(4);          % orthogonal matrix
h = Miller(0,0,1,cs);
r = vector3d.Y;
%f = fibre(h,r);
odf2 = BinghamODF(kappa,h,r,cs,ss);
Jindex_odf2 = textureindex(odf2)

figure(2)
h = [Miller(0,0,1,cs) Miller(1,0,0,cs) Miller(1,1,0,cs)];
plotPDF(odf2,h,'antipodal','silent');`

Then a problem occurred, referring to the variable h is not defined.
![figure](https://user-images.githubusercontent.com/75005172/207580899-c1accd1a-f694-4d14-9f53-32925ca61b6f.png)

Could anyone help me to fix this problem.

Thanks in advance.

All best, Yi

**Comment by kilir** [comment]
Hi,
I cannot reproduce your issue. I assume you are using the latest version of mtex?
Cheers,
Rüdiger

**Reply by caoyi0701** [reply]
Hi Rüdiger,

Thank you for your reply.
Yes, I am using the latest ver. 5.8.2.

All best, Yi
